doc/source/data/transforming-data.rst

```diff
@@ -180,12 +180,39 @@ format from your function, but ``batch_format`` should match the input of your f
                 ray.data.read_csv("s3://anonymous@air-example-data/iris.csv")
                 .map_batches(drop_nas, batch_format="pandas")
             )
+    .. tab-item:: pyarrow
+
+        .. testcode::
+
+            import pyarrow as pa
+            import pyarrow.compute as pc
+            import ray
+
+            def drop_nas(batch: pa.Table) -> pa.Table:
+                return pc.drop_null(batch)
+
+            ds = (
+                ray.data.read_csv("s3://anonymous@air-example-data/iris.csv")
+                .map_batches(drop_nas, batch_format="pyarrow")
+            )
+            
+Choosing the right batch format
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    When choosing the appropriate batch format for your ``map_batches`` operation, the primary consideration is the trade-off between convenience and performance:
+
+    * Use numpy in ``map_batches`` when your batch function needs fast numeric or tensor-style operations.
+    * Use pandas in ``map_batches`` when your batch function needs a DataFrame API, such as for tabular cleaning, joins, grouping, or row/column-wise transforms.
+    * Use pyarrow in ``map_batches`` when your batch function benefits from columnar processing, high-performance I/O, or zero-copy conversion to other systems.
+
+    Note that Ray Data uses zero-copy when the batch format matches the underlying block type (for example, Arrow blocks with ``batch_format="pyarrow"``). When they differ, Ray Data copies the data during conversion.
```

Comment:
This isn't rendering correctly: 
<img width="1095" height="368" alt="image" src="https://github.com/user-attachments/assets/397e2e68-4716-42f0-aa0d-e3c85b53b6ff" />